core/kits/assets/js/globals/global-select-behavior.js

```diff
@@ -19,7 +19,10 @@ export default class GlobalControlSelect extends Marionette.Behavior {
 		this.ui.globalPreviewsContainer.on( 'click', '.e-global__preview-item', ( event ) => this.applySavedGlobalValue( event.currentTarget.dataset.globalId ) );
 		this.ui.globalControlSelect.on( 'click', ( event ) => this.toggleSelect( event ) );
 		this.ui.manageGlobalsButton.on( 'click', () => {
-			$e.run( 'panel/global/open' ).then( () => $e.route( 'panel/global/colors-and-typography' ) );
+			const { controlType } = this.view.getGlobalMeta();
```

Comment:
Please add the full route to the `getGlobalMeta`.

Reply:
This way you can change the prefix/suffix of the tab name with no need to change the meta config

Reply:
It's better to hold the complete data, instead to use string manipulation.